nccl/tests/nccl-multi-node.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+#
+# Copyright 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+#
+
+set -e
+
+echo "'INFO '==> Staring preparation for NCCL multi-node Tests"
+
+source $WORKSPACE/libfabric-ci-scripts/nccl/common/nccl-common.sh
+
+# Number of nodes used for nccl tests, at least 3 nodes are required for ring unit test
+NUM_NODES=3
+
+# Each node has 8 GPUS
```

Comment:
We might want these scripts to be flexible to support smaller instace types as well, instead of assuming 8 GPUs/instance.

Reply:
Yes in case of less powerful instances it can be different, but for now we are using only p3dn.24xlarge instances to run unit tests and all_reduce_perf test. And that comment added to elaborate the calculation logic.